<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.walletapplication.payme.model.exceptions.EntityAlreadyExistsExceptionTest" tests="2" skipped="0" failures="0" errors="0" timestamp="2024-01-15T15:52:36" hostname="nishi" time="0.029">
  <properties/>
  <testcase name="testParameterizedConstructor()" classname="com.walletapplication.payme.model.exceptions.EntityAlreadyExistsExceptionTest" time="0.027"/>
  <testcase name="testDefaultConstructor()" classname="com.walletapplication.payme.model.exceptions.EntityAlreadyExistsExceptionTest" time="0.001"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
